datasets/lv/magnitsky_list/crawler.py

```diff
@@ -0,0 +1,66 @@
+import csv
+from typing import Dict
+import zavod.helpers as h
+from zavod import Context
+
+# Define constants for date parsing
+DATE_FORMATS = ["%Y-%m-%d", "%Y"]
+
+# Normalized country names
+NORMALIZED_COUNTRIES = {"Krievijas Federācija": "Russia"}
+
+
+def normalize_country(country: str) -> str:
+    """Normalize the country name to a standard format"""
+    return NORMALIZED_COUNTRIES.get(country, country)
```

Comment:
Ha! Finally I get to show you another mechanism :) 

We have this mechanism called data patches that's used for various data types - countries are often the most prominent. The way it works is that you can define overrides/cleanups in the YAML of the crawler metadata that are then applied inside `entity.add()` if you're trying to set a value of a specific type. Have a look at: 

https://github.com/opensanctions/opensanctions/blob/main/datasets/_externals/wikidata.yml#L86-L654

This example is a bit bonkers, but shows the principle. Wanna try?